### DISCUSSION TITLE
Small typo in the Documentation

### DISCUSSION THREAD

**Discussion by sabamimi** [MDeLuise/plant-it#194] [General]
In the documentation :
https://docs.plant-it.org/0.7.1/server-installation/#complete-example
Under # Server config, there is a small misspelled word :  "TRAFLE_KEY=" intead of TREFLE_KEY=
I did copy the complete example and it works first time around except for this part.
(Very nice application, I am still in the discovery phase)

**Comment by MDeLuise** [comment]
Hi @sabamimi,
really thanks for pointing this out, I updated the documentation!
